raiden/raiden_service.py

```diff
@@ -656,8 +656,8 @@ def get_block_number(self) -> BlockNumber:
         assert self.wal, f"WAL object not yet initialized. node:{self!r}"
         return views.block_number(self.wal.state_manager.current_state)
 
-    def on_message(self, message: Message, is_light_client: bool = False):
-        self.message_handler.on_message(self, message, is_light_client)
+    def on_message(self, message: Message, address: Address, is_light_client: bool = False):
```

Comment:
`address` should make reference to either the sender or the receiver through its param name